packages/inventory/src/services/inventory-level.ts

```diff
@@ -21,6 +21,7 @@ export default class InventoryLevelService extends TransactionBaseService {
     CREATED: "inventory-level.created",
     UPDATED: "inventory-level.updated",
     DELETED: "inventory-level.deleted",
+    DELETED_BY_LOCATION: "inventory-level.deleted-by-location",
```

Comment:
suggestion: rename to BULK_DELETED and allow the event data to be dynamic based on the query. I.e.: Event data could be: `{ location_id: ... }` but `{ inventory_item_id: ... }` would also be valid. 

This way we don't have to create `DELETED_BY_X` for every deletion dimension. People who want to build synchronization could simply listen to `DELETED` and `BULK_DELETED` events.

@adrien2p, @olivermrbl - curious to hear your thoughts on this too?

Reply:
Could you give an example maybe? :)

Reply:
I would be fine with that too -- would just have to be documented.

Reply:
Eventually, second options is 

```ts
const { raw } = levelRepository.delete({ location_id: locationId })
const events = raw.map(level => ({ eventName: InventoryLevel.Events.DELETED, data: { id: level.id })
await this.eventBusService_.withTransaction(manager).emit(events)
```

Wdyt @srindom ?

Reply:
Going for the pure deleted event, where do you want it documented @srindom?

Reply:
cc: @shahednasser, can we document this behavior in the events reference :)

Reply:
@srindom sure, once it's merged I'll include this behavior in the events reference (creating a linear issue for it) 👍🏻